realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -735,6 +735,93 @@ public void onError(ObjectServerError error) {
         });
     }
 
+    @Test
+    @RunTestInLooperThread
+    public void revokeOffer() {
+        // createOffer validates that the offer is actually in the __management Realm.
```

Comment:
`makeOffer` called inside `createOffer` doesn't return until the offer is succesfully created and ready.